open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -35,66 +39,42 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
 
         <dependency>
-            <groupId>org.springframework.security</groupId>
```

Comment:
there's a lot of reordering in this file which makes it rather hard to review. Will return to it, but also recommend we do a before and after of 'mvn dependency:tree' and check the results look good.